### PR TITLE
update rubygems to v2.6.14

### PR DESCRIPTION
**Pull request by walf443**
## What was the end-user problem that led to this PR?

rubygems v2.6.13 has important security issues.
So, It's better to update v2.6.14 to rubygems.

http://blog.rubygems.org/2017/10/09/2.6.14-released.html

### CODE REVIEW

**Comment by walf443** [comment]
hmm. for supporting ruby1.8, it seems that it should update psych deps that support YAML.safe_load

File: .travis.yml [hunk 1]
```diff
@@ -48,7 +48,8 @@ matrix:
       env: RGV=master
     # 1.x mode (we want to keep stuff passing in 1.x mode for now)
     - rvm: 2.4.2
-      env: RGV=v2.6.13 BUNDLER_SPEC_SUB_VERSION=1.98
+      env: RGV=v2.6.14 BUNDLER_SPEC_SUB_VERSION=1.98
+    # RGV 2.6.13 is the final version that run with ruby1.8.x
```
Comment (segiddins) [inline .travis.yml]: this is not true, 2.6.14 supports 1.8.7

**Comment by hsbt** [comment]
psych-2.0.0 didn't support Ruby 1.8.7. https://github.com/ruby/psych/blob/e04307d69605b5daa650a6df8016c4a34296ab1b/Rakefile#L28

**Comment by walf443** [comment]
Is it better to add dependency larger than psych 2.0.0?

**Comment by segiddins** [comment]
Psych 2 not supporting ruby 1.8.7 is totally fine, RubyGems still works on 1.8.7 (as I tested last night before releasing 2.6.14)

**Comment by walf443** [comment]
https://github.com/bundler/bundler/pull/6088 may fixed this problem.

**Comment by walf443** [comment]
waiting https://github.com/rubygems/rubygems/pull/2034 is merged.

**Comment by mikegee** [comment]
https://github.com/rubygems/rubygems/pull/2034 is merged now

**Comment by walf443** [comment]
https://github.com/bundler/bundler/pull/6142 created. So closed

**Pull request closed without merging**